plexapi/video.py

```diff
@@ -925,13 +962,48 @@ def _loadData(self, data):
         self.thumbAspectRatio = data.attrib.get('thumbAspectRatio')
         self.viewOffset = utils.cast(int, data.attrib.get('viewOffset', 0))
         self.year = utils.cast(int, data.attrib.get('year'))
-
+        
     @property
     def locations(self):
         """ This does not exist in plex xml response but is added to have a common
             interface to get the locations of the clip.
 
-            Retruns:
+            Returns:
                 List<str> of file paths where the clip is found on disk.
         """
         return [part.file for part in self.iterParts() if part]
+
+    def section(self):
+        """Return the :class:`~plexapi.library.LibrarySection` this item belongs to."""
+        # Clip payloads currently do not contain 'librarySectionID'.
+        # Return None to avoid unnecessary attribute lookup attempts.
```

Comment:
`Clip` is also used for `PhotoSection` videos which do have a `librarySectionID`.

Reply:
`# Clip payloads may not contain 'librarySectionID'.` ?

Reply:
Move `section()` from `Clip` to `Extra`. Then other clips will still work, and extras will return `None`.

But maybe it's possible to get the `librarySectionID` for extras from the parent. The extras belong to a movie, and that movie belongs to a library.